.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+
+[*.md]
+# Trailing whitespace is important for Markdown
+trim_trailing_whitespace = false
+
+[*.{kt,kts}]
+max_line_length = 200
+
+[*.java]
+indent_size = 4
```

Comment:
I think it should be used for [*]

Reply:
I don't think so. We use 2 spaces for yaml configs, for example

Reply:
Oh, whatever you like. Frankly speaking, I have never used Kaspresso :)
It was just recently mentioned when we were discussing a talk for https://heisenbug-piter.ru/ , and I noticed editorconfig might be helpful here

Reply:
I've added `indent_size = 2` for yaml